src/api/messagesFlags.js

```diff
@@ -1,14 +1,16 @@
-import { apiPost, Auth } from './apiFetch';
+/* @flow */
+import { Auth } from '../types';
+import { apiPost } from './apiFetch';
 
 export default (
   auth: Auth,
   messages: number[],
   op: string,
   flag: string,
-): number[] =>
+): mixed =>
```

Comment:
If it is possible to declare the response more concretely than `mixed` that would be important to do.